src/main/scala/com/softwaremill/realworld/Main.scala

```diff
@@ -1,35 +1,34 @@
 package com.softwaremill.realworld
 
-import org.slf4j.LoggerFactory
+import com.softwaremill.realworld.articles.ArticlesEndpoints
 import sttp.tapir.server.interceptor.log.DefaultServerLog
 import sttp.tapir.server.ziohttp.{ZioHttpInterpreter, ZioHttpServerOptions}
+import zio.Cause.Die
 import zio.http.{HttpApp, Server, ServerConfig}
-import zio.{Console, ExitCode, Scope, Task, ZIO, ZIOAppArgs, ZIOAppDefault}
+import zio.{Cause, Console, ExitCode, Scope, StackTrace, Task, ZIO, ZIOAppArgs, ZIOAppDefault}
 
 object Main extends ZIOAppDefault:
-  val log = LoggerFactory.getLogger(ZioHttpInterpreter.getClass.getName)
-
   override def run: ZIO[Any with ZIOAppArgs with Scope, Any, Any] =
     val serverOptions: ZioHttpServerOptions[Any] =
       ZioHttpServerOptions.customiseInterceptors
         .serverLog(
           DefaultServerLog[Task](
-            doLogWhenReceived = msg => ZIO.succeed(log.debug(msg)),
-            doLogWhenHandled = (msg, error) => ZIO.succeed(error.fold(log.debug(msg))(err => log.debug(msg, err))),
-            doLogAllDecodeFailures = (msg, error) => ZIO.succeed(error.fold(log.debug(msg))(err => log.debug(msg, err))),
-            doLogExceptions = (msg: String, ex: Throwable) => ZIO.succeed(log.debug(msg, ex)),
+            doLogWhenReceived = msg => ZIO.debug(msg),
```

Comment:
why not `.logDebug`? Also, there are some default logging options (`ZioHttpServerOptions.defaultServerLog`), aren't they sufficient?

Reply:
Ah right, they are sufficient, didn't know it exists.

Reply:
I've used the default configuration, but it surfaced a problem - default zio logger can log only >= `INFO` messages. So I've also changed default logger to logback, as it is more configurable.